huntsman/observatory.py

```diff
@@ -104,6 +109,15 @@ def initialize(self):
             self.logger.debug("Connecting to autoguider")
             self.autoguider.connect()
 
+    def past_midnight(self):
```

Comment:
Based on how this is used below, this should be a property instead of a method.

Also, `past_midnight` seems a little ambiguous to me. I wonder if we could get more clarity with our conditionals by having a bunch of predicate properties: `is_evening`, `is_night`, `is_morning`, `is_day`.  We observe when it `is_night`, we calibrate when it `is_evening` or `is_morning`, we sleep when it `is_day`.